src/processor/operator/simple/use_database.cpp

```diff
@@ -14,5 +14,11 @@ std::string UseDatabase::getOutputMsg() {
     return "Used database successfully.";
 }
 
+std::string UseDatabasePrintInfo::toString() const{
+    std::string result = "Database Name: ";
```

Comment:
Let's change it to say `"Database: "` instead of `"Database Name: "` to match the other database physical operators

Reply:
How do I run the clang format? From the error message on the test it seems like it's trying to push to my repository but failing

Reply:
You can run clang format locally using the script I linked. If you are having trouble using the script you can ask Chang on discord for help figuring out how to run the script 👍